Common/UnitDefinitions/DynamicViscosity.json

```diff
@@ -79,7 +79,19 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "lbf·s/ft²" ]
+          "Abbreviations": ["lbf·s/ft²"]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundFootSecond",
+      "PluralName": "PoundsFootSecond",
+      "FromUnitToBaseFunc": "x * 1.4881639",
+      "FromBaseToUnitFunc": "x / 1.4881639",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/ft-s" ]
```

Comment:
This is debatable, but the convention in Units.NET is to favor ft·s instead of ft-s, for consistency across units. We can make an exception if this other form is clearly more widely used, let me know.

We can also add both abbrevation forms if it adds value, but the first item in the array is the one used by ToString() by default so we need to choose a primary abbrevation in that case.